src/ngraph/op/fused/shuffle_channels.cpp

```diff
@@ -23,18 +23,18 @@ using namespace ngraph;
 
 constexpr NodeTypeInfo op::ShuffleChannels::type_info;
 
-op::ShuffleChannels::ShuffleChannels(const Output<Node>& data, const int axis, const size_t groups)
+op::ShuffleChannels::ShuffleChannels(const Output<Node>& data, const int axis, const size_t group)
```

Comment:
Noted in passing:
Would it be a problem to make `groups` a `uint64_t`? The size of `size_t` varies from platform to platform.
Same with `axis` -- it should be `int64_t`

Reply:
There is nothing wrong with size_t but there is a problem when people assume that size_t is 64 bits. You should never mix size_t with other types like int64_t, if you use size_t use size_t everywhere it touches.

Reply:
People assume that a lot of places, and the compiler isn't warning about it when it's not true.